### ISSUE TITLE
No issue, but thanks!

### ISSUE THREAD

**Issue by Korokun** [m-mayday/game-translations#2]
Any other games you have in mind you want to work on? And need an editor? Also thanks!

**Comment by m-mayday** [comment]
Thanks! I plan to start working on Atelier Elie GB soon. I don't want to form a team or anything like that, since I don't want it to become a chore, and I like to work on it at my own pace.
That being said, I'll take any help I can get, especially when it comes to the quality of the translation. What does an editor entail, proofreading?

**Comment by Korokun** [comment]
Yes exactly. Just give me your discord username and we'll talk in DMs. I'm honestly down for whatever as long as it helps make progression. I'm MrCatford. Either way I'll find a way to chat and go from there.

**Comment by marielin** [comment]
Hey! I don't want to clog up your GitHub issues, so I figure I'll add onto this one: I've been waiting for a translation of Atelier Marie GB for a long time now and I'm so incredibly excited that you've made one. Thank you so much for your hard work, it is deeply appreciated!

**Comment by Korokun** [comment]
Hey it's been over a year! I saw that you did finish Elie! Congrats! Any other future projects in mind?

**Comment by m-mayday** [comment]
> Hey it's been over a year! I saw that you did finish Elie! Congrats! Any other future projects in mind?

Hey, thanks! I'm working on something at the moment but I'm not ready to announce it yet.
